panel/io/convert.py

```diff
@@ -42,17 +42,17 @@
 BOKEH_VERSION = base_version(bokeh.__version__)
 PY_VERSION = base_version(__version__)
 PYODIDE_VERSION = 'v0.25.0'
-PYSCRIPT_VERSION = '2024.7.1'
+PYSCRIPT_VERSION = '0.5.1'
 WHL_PATH = DIST_DIR / 'wheels'
 PANEL_LOCAL_WHL = WHL_PATH / f'panel-{__version__.replace("-dirty", "")}-py3-none-any.whl'
 BOKEH_LOCAL_WHL = WHL_PATH / f'bokeh-{BOKEH_VERSION}-py3-none-any.whl'
 PANEL_CDN_WHL = f'{CDN_DIST}wheels/panel-{PY_VERSION}-py3-none-any.whl'
 BOKEH_CDN_WHL = f'{CDN_ROOT}wheels/bokeh-{BOKEH_VERSION}-py3-none-any.whl'
 PYODIDE_URL = f'https://cdn.jsdelivr.net/pyodide/{PYODIDE_VERSION}/full/pyodide.js'
 PYODIDE_PYC_URL = f'https://cdn.jsdelivr.net/pyodide/{PYODIDE_VERSION}/pyc/pyodide.js'
-PYSCRIPT_CSS = f'<link rel="stylesheet" href="https://pyscript.net/releases/{PYSCRIPT_VERSION}/core.css" />'
+PYSCRIPT_CSS = f'<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@pyscript/core@{PYSCRIPT_VERSION}/dist/core.css" />'
```

Comment:
I would recommend sticking to the urls explained in the pyscript documentation. See https://docs.pyscript.net/2024.7.1/user-guide/first-steps/.

I don't know what the jsdelivr urls are fore and why they are versioned differently.

Reply:
2024.7.1 presumably does not have the fix @WebReflection implemented.